src/20240306_AaveV3Ethereum_AaveLiquidityCommiteeFunding/AaveLiquidityCommiteeFunding.md

```diff
@@ -0,0 +1,102 @@
+
+| Objective         | Category       | Budget   |
+| ----------------- | -------------- | -------- |
+| Liquidity         | Peg Resilience | 360k GHO |
+| DeFi Integrations | Create Utility | 140k GHO |
+| Total             |                | 500k GHO |
+
+It is important to note that this estimation excludes efforts to drive GHO liquidity or integrations beyond Ethereum.
+
+# Performance Indicators
+
+The below details some high level GHO metrics we propose tracking:
+
+| Description                    | Optimal Value                                                                      |
+| ------------------------------ | ---------------------------------------------------------------------------------- |
+| TVL DEX Liquidity Pools        | 50M                                                                                |
+| TVL in Utility Liquidity Pools | 15M excl. stkGHO                                                                   |
+| DEX Liquidity Composition      | < 50% GHO (< 33% for 3pools)                                                       |
+| Swap Price Impact $5M Swap     | < 0.10% (GHO to USDC)                                                              |
+| Annualised Peg Volatility      | < 5.00%                                                                            |
+| Price level for > 90% time     | $0.995 using [Redstone Medium Price](https://app.redstone.finance/#/app/token/GHO) |
+
+Please note, each of the above targets has external dependencies beyond the control of the ALC. The above table serves as a North Star for the ALC to strive towards over the next 3 months on the assumption the Borrow Cap is continually increased beyond the current 35M Bucket Size. Having measurable targets provides a clear direction and goal to achieve.
+
+The [stkGHO implementation](https://app.aave.com/staking), [USDe integration](https://app.ethena.fi/liquidity) and coordinated efforts to attract users to acquire GHO are all great examples of how growth initiatives drive GHO utility, adoption and have a positive impact the spot price.
+
+Each of the above metrics, plus more, will be shown on the TokenLogic analytics dashboard.
+
+# Specification
+
+Swap the following assets for GHO:
+
+- Withdraw 250k Aave V2 USDT from the collector
+- Withdraw 250k Aave V2 USDC from the collector
+- Swap 250k USDT withdrawn above for GHO
+- Swap 250k USDC withdrawn above for GHO
+
+Create an allowance of 500k GHO to ALC SAFE.
+
+ALC SAFE: `0x205e795336610f5131Be52F09218AF19f0f3eC60`
+
+# Disclosure
+
+TokenLogic and karpatkey receive no payment for this proposal. TokenLogic and karpatkey are both delegates within the Aave community.
+
+# Next Steps
+
+1. Gather feedback from the community.
+1. If consensus is reached on this ARFC, escalate this proposal to the Snapshot stage.
+1. If Snapshot outcome is YAE, escalate this proposal to AIP stage
+
```

Comment:
@Luigy-Lemon Missing references to payload implementation, tests and snapshot details